python/QC3_Plotter.py

```diff
@@ -0,0 +1,310 @@
+
+import pandas as pd
+import xlrd
+from array import array
+import numpy as np
+import ROOT as r
+
+import argparse
+from optparse import OptionParser
+
+parser = argparse.ArgumentParser()
+parser.add_argument('filename', nargs='+')
+args = parser.parse_args()
+
+par1_Short= array( 'f' )
+par1_Long= array( 'f' )
+Err_par1_Short= array( 'f' )
+Err_par1_Long= array( 'f' )
+errX_Short=array('f')
+errX_Long=array('f')
+Detector_Short=array('f')
+Detector_Long=array('f')
+
+for f in args.filename:
```

Comment:
Why is this file executing all this line-by-line?  The other python scripts for a given QC script make use of `python/Produce_Config_File.py` to create the config file.

The `genericPlotter` will set the style automatically, and will also do 1D fitting if the user configures the fit information correctly.

This is unnecessary code.  Please change to be more in line with other scripts.

Use the existing tools, don't reinvent them.